themes/Anari/css/style.css

```diff
@@ -10,6 +10,12 @@ body {
   line-height: 1.4;
 }
 
+:root {
```

Comment:
Can we move this to after the meyer reset. Ideally the reset should happen first and then our custom styling afterward.

Reply:
Yes!